### PR TITLE
Add optional string of content to check in response

### PR DESCRIPTION
**Pull request by Brunty**
Format of the `.cigar.json` file as follows:

```
[
  {
    "url": "http://httpbin.org/status/418",
    "status": 418,
    "content": "teapot"
  },
  {
    "url": "http://httpbin.org/status/200",
    "status": 200
  },
  {
    "url": "http://httpbin.org/status/304",
    "status": 304
  },
  {
    "url": "http://httpbin.org/status/500",
    "status": 500
  }
]
```

### CODE REVIEW

File: src/Parser.php [hunk 1]
```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```
Comment (Brunty) [inline src/Parser.php]: This throws errors if we don't have 3 parts, but due to not showing them in `bin/cigar` it's never seen 👀

Comment (asgrim) [inline src/Parser.php]: Bit of a naive split here; doesn't account for content search with spaces, e.g. `http://httpbin.org/status/418 418 "teapot blah"`

Comment (Brunty) [inline src/Parser.php]: yeah, I'll have to re-think the splitting

Comment (Brunty) [inline src/Parser.php]: I had considered making the file a `json` file for easier parsing.

Comment (asgrim) [inline src/Parser.php]: That would be reasonable (also, easier for adding new fields later), although it does do away with the insanely simple setup at the moment... It's a tradeoff I guess!

Comment (Brunty) [inline src/Parser.php]: Yeah, but I think with the addition of content to search for, we're moving a little past simple - I'll give it some thought.

Comment (asgrim) [inline src/Parser.php]: yup indeed; depends what you want the scope of the project to be; it's your call as to exactly how much scope creep you want; was just an idea :D

Comment (Brunty) [inline src/Parser.php]: Moved to `json` - still feels fairly simple and means new things are *way* easier to add in the future :)

File: README.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ Install via composer:
 
 ## To use
 
-Create a `.cigar` file that contains a newline separated list with the URLs you want to check, followed by a space, followed by the expected status code:
+Create a `.cigar` file that contains a newline separated list with the URLs you want to check, followed by a space, followed by the expected status code and optionally, a quoted string of content you want to see within the response body.
 
 ```
-http://httpbin.org/status/418 418
+http://httpbin.org/status/418 418 "teapot"
```
Comment (asgrim) [inline README.md]: Example is wrong here now :/

Comment (Brunty) [inline README.md]: Yeah, needs fixing, I ran out of time on my lunchbreak.

Tests are also breaking on CI but working locally, so I'll debug that later.

**Comment by asgrim** [comment]
:+1:
